backend/shork/src/test/kotlin/TestInternalShork.kt

```diff
@@ -42,27 +48,26 @@ internal class TestInternalShork {
 
     @Test
     fun `Player can win`() {
```

Comment:
Other test names are not capitalized, this one should start with a lowercase character too